channel/server.go

```diff
@@ -107,6 +107,46 @@ func (server *Server) Initialise(work chan func(), dbuser, dbpassword, dbaddress
 		prometheus.MustRegister(common.MetricsGauges["player_count"])
 	}
 
+	if _, ok := common.MetricsCounters["monster_kills_total"]; !ok {
+		common.MetricsCounters["monster_kills_total"] = prometheus.NewCounterVec(prometheus.CounterOpts{
+			Name: "monster_kills_total",
+			Help: "Total number of monsters killed",
+		}, []string{"channel", "world"})
```

Comment:
I think it would be useful to also have character ID be a label as you could plot the rates and outliers would probably be hackers